v2.0/demo-json-support.md

```diff
@@ -0,0 +1,186 @@
+~~~
+
+## Step 4. Create a database and grant privileges
+
+As the `root` user, open the [built-in SQL client](use-the-built-in-sql-client.html):
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql --insecure
+~~~
+
+Next, create a database called `jsonb_test`:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE DATABASE jsonb_test;
+~~~
+
+Set the database as the default:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SET DATABASE = jsonb_test;
+~~~
+
+Then [grant privileges](grant.html) to the `maxroach` user:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> GRANT ALL ON DATABASE jsonb_test TO maxroach
+~~~
+
+## Step 5. Create a table
+
+In the [built-in SQL client](use-the-built-in-sql-client.html), create a table called `programming`:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE TABLE rprogramming (
+    id UUID DEFAULT uuid_v4()::UUID PRIMARY KEY,
+    posts JSONB
+  );
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW CREATE TABLE programming;
+~~~
+~~~
++--------------+-------------------------------------------------+
+|    Table     |                   CreateTable                   |
++--------------+-------------------------------------------------+
+| programming  | CREATE TABLE programming (                      |
+|              |     id UUID NOT NULL DEFAULT uuid_v4()::UUID,   |
+|              |     posts JSON NULL,                            |
+|              |     CONSTRAINT "primary" PRIMARY KEY (id ASC),  |
+|              |     FAMILY "primary" (id, posts)                |
+|              | )                                               |
++--------------+-------------------------------------------------+
+~~~
+
+## Step 6. Run the code
+
+Now that you have a database, user, and a table, let's run code to insert rows into the table.
+
+The code queries the [Reddit API](https://www.reddit.com/dev/api/) for posts in [/r/programming](https://www.reddit.com/r/programming/). The Reddit API only returns 25 results per page; however, each page returns an `"after"` string that tells you how to get the next page. Therefore, the program does the following in a loop:
+
+1. Makes a request to the API
+2. Inserts the results into the table and grabs the `"after"` string
+3. Uses the new `"after"` string as the basis for the next request
+
+Download the <a href="https://raw.githubusercontent.com/cockroachdb/docs/master/_includes/json/json-sample.go" download><code>json-sample.go</code></a> file, or create the file yourself and copy the code into it:
+
+{% include copy-clipboard.html %}
+~~~ go
+{% include json/json-sample.go %}
+~~~
+
+From a new terminal window, navigate to your sample code file and run it:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ go run json_sample.go
+~~~
+
+The program will take a few minutes to insert rows into your table. If you want to verify that rows of data were inserted into your table,  use the [built-in SQL client](use-the-built-in-sql-client.html):
```

Comment:
I'd probably end here with `The program will take a while to finish, but you can start querying the data right away.

Reply:
done